jac-uk/admin
# Feat/2387 improve eligibility report v2

## What's included?
closed #2387 

- Admins can make separate recommendations for Stat criteria, PJE and RLoS on Legal exercises
- Admins can view candidate responses where they haven't met PJE or RLoS on Legal exercises
- Admins can add separate comments about their separate recommendations for Stat criteria, PJE and RLoS on Legal exercises

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- New Legal exercise: Can add recommendation and reasons for individual eligibility issue
    - Go to the preview link: https://jac-admin-develop--pr2390-feat-2387-improve-el-s0bfjq10.web.app/exercise/S9V9R15MGRjCDHdcRQRZ/reports/eligibility-issues
    - Click `Refresh` button to prepare eligibility issue data.
    - It should appear recommendation dropdown lists for each eligibility issue.
    - If the recommendation being selected, it should appears `Reasons not satisfied` box.
    - It should display candidate comments for `Previous Judicial Experience`  and `Reasonable length of service` issue
    - `Recommendation` and `Reasons not satisfied` changes can be saved.
- Existing Legal exercise: The original overall recommendation will become the recommendation of individual eligibility issue
    - Go to the preview link:  https://jac-admin-develop--pr2390-feat-2387-improve-el-s0bfjq10.web.app/exercise/pmhY2nNEQFg4OXzSDPYw/reports/eligibility-issues 
    - It should appear recommendation dropdown lists for each eligibility issue.
- Non Legal exercise: Stay the same as before, can add  an overall recommendation for eligibility issues
    - Go to the preview link:  https://jac-admin-develop--pr2390-feat-2387-improve-el-s0bfjq10.web.app/exercise/OTzmqJ0KOdyT4h1azw7X/reports/eligibility-issues
    - Click `Refresh` button to prepare eligibility issue data.
    - It should appear one overall recommendation dropdown lists for eligibility issues.
    
## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Include screen grabs, video demo, notes etc.

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required
- Permissions have been added / updated. Details:

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
